static/redfish/v1/JsonSchemas/OemLogEntry/index.json

```diff
@@ -22,6 +22,14 @@
                 }
             },
             "properties": {
+                "InitiatedBy": {
+                    "description": "Id of the external user who initiated the dump.",
+                    "longDescription": "The URI of the session of the external user who initiated the dump. This is intended to be populated for user-initiated dumps.",
```

Comment:
Id of the external user who created the LogEntry. Please dont mention 'dump' in the schema descriptions as this is a LogEntry schema which can be used for many other use-cases other than dumps.